packages/cli/snapshots/integration/generators/import-lb3-models.integration.snapshots.js

```diff
@@ -11,13 +11,7 @@ exports[`lb4 import-lb3-models imports CoffeeShop model from lb3-example app 1`]
 import {Entity, model, property} from '@loopback/repository';
 
 @model({
-  settings: {
-    strict: false,
-    forceId: false,
-    replaceOnPUT: true,
-    validateUpsert: true,
-    idInjection: true
-  }
+  settings: {strict: false, forceId: false, validateUpsert: true, idInjection: true}
```

Comment:
The following settings are no longer imported because they are inherited:

- `replaceOnPUT`

Please note that `forceId`, `idInjection` and `validateUpsert` are explicitly configured in coffee-shop's model JSON file, therefore they are preserved in the generated output. You can compare this line with `User` settings below https://github.com/strongloop/loopback-next/pull/4737/files#diff-e9be610fe11bc2c954313d9283d0eda3R128-R137, where only `strict` + custom settings are generated.